packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1401,12 +1727,28 @@ class Scaffold extends StatefulWidget {
   ///   Widget build(BuildContext context) {
   ///     return Center(
   ///       child: ElevatedButton(
-  ///         child: Text('SHOW A SNACKBAR'),
+  ///         child: Text('SHOW BOTTOM SHEET'),
   ///         onPressed: () {
-  ///           Scaffold.of(context).showSnackBar(
-  ///             SnackBar(
-  ///               content: Text('Have a snack!'),
-  ///             ),
+  ///           Scaffold.of(context).showBottomSheet<void>(
+  ///             (BuildContext context) {
+  ///               return Container(
+  ///                 height: 200,
+  ///                 color: Colors.amber,
+  ///                 child: Center(
+  ///                   child: Column(
+  ///                     mainAxisAlignment: MainAxisAlignment.center,
+  ///                     mainAxisSize: MainAxisSize.min,
+  ///                     children: <Widget>[
+  ///                       const Text('BottomSheet'),
+  ///                       ElevatedButton(
+  ///                         child: const Text('Close BottomSheet'),
+  ///                         onPressed: () => Navigator.pop(context),
```

Comment:
I'd prefer it if we reserved `=>` for one-liners that returned a value (not void). Here and elsewhere.